pkg/upgrade/upgrader/jiva_upgrade.go

```diff
@@ -117,35 +118,67 @@ func getControllerPatchDetails(d *appsv1.Deployment) (
 	return patchDetails, nil
 }
 
-func getReplica(replicaLabel, namespace string) (*replicaDetails, error) {
-	replicaObj := &replicaDetails{}
-	deployObj, err := getDeployment(replicaLabel, namespace)
-	if err != nil {
-		return nil, errors.Wrapf(err, "failed to get replica deployment")
-	}
-	if deployObj.Name == "" {
-		return nil, errors.Errorf("missing deployment name for replica")
-	}
-	replicaObj.name = deployObj.Name
-	version, err := getOpenEBSVersion(deployObj)
+func getPatchDetailsForDeploy(pvName string, deployObj *appsv1.Deployment) (*replicaPatchDetails, error) {
```

Comment:
n_: As this is specific to Replica Patch. Can we change this to `getPatchDetailsForReplicaDeploy`